src/mobile/package.json

```diff
@@ -8,6 +8,8 @@
     "ios": "expo start --ios",
     "web": "expo start --web",
     "eject": "expo eject",
+    "eslint": "eslint --config .eslintrc.js \"**/*.js\"",
```

Comment:
Can you include `jsx` for /mobile too

Reply:
Done. Thanks for reviewing this so thoroughly. I actually missed a lot of stuff and wouldn't have seen them if you and everyone else didn't review